account_payment_order/README.rst

```diff
@@ -0,0 +1,97 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+
+Payment order
+============================
+
+This module brings back the payment.order from v8.
+
+This module also provide an infrastructure to export payment orders.
+It includes some bug fixes and obvious enhancements to payment orders that will hopefully land in offical addons one
+day.
+This technical module provides the base infrastructure to export payment orders
```

Comment:
This text was specific for the account_banking_payment_export. I think we can remove it.